sdks/python/apache_beam/examples/inference/pytorch_image_classification.py

```diff
@@ -74,7 +74,7 @@ def parse_known_args(argv):
       '--input',
       dest='input',
       default='gs://apache-beam-ml/testing/inputs/'
-      'it_mobilenetv2_imagenet_validation_inputs.txt',
```

Comment:
was this intentional?

Reply:
No. I modified the wrong file. Good catch. Fied.